build/commands/lib/config.js

```diff
@@ -103,108 +113,108 @@ const Config = function () {
   this.scriptDir = path.join(this.rootDir, 'scripts')
   this.srcDir = path.join(this.rootDir, 'src')
   this.chromeVersion = this.getProjectVersion('chrome')
-  this.chromiumRepo = getNPMConfig(['projects', 'chrome', 'repository', 'url'])
+  this.chromiumRepo = getDotenvConfig(['projects', 'chrome', 'repository', 'url'])
   this.braveCoreDir = braveCoreDir
   this.buildToolsDir = path.join(this.srcDir, 'build')
   this.resourcesDir = path.join(this.rootDir, 'resources')
   this.depotToolsDir = path.join(this.braveCoreDir, 'vendor', 'depot_tools')
   this.defaultGClientFile = path.join(this.rootDir, '.gclient')
-  this.gClientFile = process.env.BRAVE_GCLIENT_FILE || this.defaultGClientFile
-  this.gClientVerbose = getNPMConfig(['gclient_verbose']) || false
-  this.targetArch = getNPMConfig(['target_arch']) || process.arch
-  this.targetOS = getNPMConfig(['target_os'])
-  this.targetEnvironment = getNPMConfig(['target_environment'])
+  this.gClientFile = getDotenvConfig.BRAVE_GCLIENT_FILE || this.defaultGClientFile
```

Comment:
Does this work? This one wasn't being loaded via the NPM config before, it was using `processs.env` before.

(There are a few more instances of this later in this file.)

Reply:
ooh, sloppy iterative find+replace on my part. That definitely won't work. Thanks for catching.

I didn't realize we were actually using `process.env` for some of these (including secrets) in the first place!

Reply:
fixed